api/Trace/Span.php

```diff
@@ -55,5 +56,7 @@ public function setSpanStatus(string $code, ?string $description = null): Span;
      */
     public function end(int $timestamp = null): Span;
 
+    public function getSpanContext(): SpanContext;
```

Comment:
This is defined no the Java Span API, and is needed due to where active spans and such are stored now. E.g. within `TraceContextPropagator::inject`. I did also notice there is `Span#getContext()` that returns a clone of the span context, possibly could just replace that method with this one.

Reply:
Yeah replacing `Span#getContext()` with this sounds a bit cleaner to me.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#get-context

Looks like we should do the opposite, my as well use the name the spec suggests.